source/elements/oneMKL/source/domains/rng/device_routines/device-distributions.rst

```diff
@@ -0,0 +1,95 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_distributions:
+
+Device Distributions
+====================
+
+
+oneMKL RNG routines are used to generate random numbers with different types of distribution. Each function group is 
+introduced below by the type of underlying distribution and contains a short description of its functionality, as well 
+as specifications of the call sequence and the explanation of input and output parameters. The Device Continuous 
+Distribution Generators table and Device Discrete Distribution Generators table list the random number generator routines 
+with data types and output distributions, and sets correspondence between data types of the generator routines and the 
+basic random number generators.
+
+**Device Continuous Distribution Generators**
+
+.. list-table::
+   :header-rows: 1
+
+   * -     Type of Distribution
+     -     Data Types
+     -     BRNG Data Type
+     -     Description
+   * -     :ref:`onemkl_device_rng_uniform_continuous`
+     -     s, d
+     -     s, d
+     -     Uniform continuous distribution on the interval [``a,b``)
+   * -     :ref:`onemkl_device_rng_uniform_gaussian`
+     -     s, d
+     -     s, d
+     -     Normal (Gaussian) distribution
+   * -     :ref:`onemkl_device_rng_uniform_exponential`
+     -     s, d
+     -     s, d
+     -     Exponential distribution
+   * -     :ref:`onemkl_device_rng_uniform_lognormal`
+     -     s, d
+     -     s, d
+     -     Lognormal distribution
+
+
+**Device Discrete Distribution Generators**
+
+.. list-table::
+   :header-rows: 1
+
+   * -     Type of Distribution
+     -     Data Types
+     -     BRNG Data Type
+     -     Description
+   * -     :ref:`onemkl_device_rng_uniform_discrete`
+     -     i
+     -     s
+     -     Uniform discrete distribution on the interval [``a,b``)
+   * -     :ref:`onemkl_device_rng_bits`
+     -     i
+     -     i
+     -     Bits of underlying BRNG integer recurrence
+   * -     :ref:`onemkl_device_rng_uniform_bits`
+     -     i
+     -     i
+     -     Uniformly distributed bits in 32-bit chunks
```

Comment:
Can we merge two lines? Like "Uniformly distributed bits in 32/64-bit chunks"

Reply:
Sure. I can make it here and in DevRef if everyone is ok with that

Reply:
Done